admin_guide/manage_nodes.adoc

```diff
@@ -415,32 +415,34 @@ anti-affinity can be set to either required or preferred.
 
 Using the `*docker-registry*` pod as an example, the first step in enabling
 this feature is to set the `*scheduler.alpha.kubernetes.io/affinity*` on the
-pod. Since this pod uses a deployment configuration, the most appropriate
-place to add the annotation is to the pod template's metadata.
-
-----
-$ oc edit dc/docker-registry -o yaml
-
-...
-  template:
-    metadata:
-      annotations:
-        scheduler.alpha.kubernetes.io/affinity: |
-          {
-            "podAntiAffinity": {
-              "requiredDuringSchedulingIgnoredDuringExecution": [{
-                "labelSelector": {
-                  "matchExpressions": [{
-                    "key": "docker-registry",
-                    "operator": "In",
-                    "values":["default"]
-                  }]
-                },
-                "topologyKey": "kubernetes.io/hostname"
-              }]
-            }
-          }
-----
+pod. 
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: with-pod-antiaffinity
+spec:
+  affinity:
+    podAntiAffinity: <1>
+      preferredDuringSchedulingIgnoredDuringExecution: <2>
+      - weight: 100 <3>
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: docker-registry <4>
+              operator: In <5>
+              values: 
+              - default
+          topologyKey: kubernetes.io/hostname
+----
+
+<1> Stanza to configure pod anti-affinity.
+<2> Defines a preferred rule.
+<3> Specifies a weight for a preferred rule. The node that with highest weight is preferred.
+<4> The key and value (label) that must be matched to apply the rule.
```

Comment:
This annotation confused me until I read annotation 5. I think it should be something like "Description of the pod label that determines when the anti-affinity rule applies. Specify a key and value for the label."
( also might attach it to the line that contains matchExpression or give annotations for both the key and value lines.)